spec/reek/cli/command/todo_list_command_spec.rb

```diff
@@ -37,6 +37,40 @@
         result = command.execute
         expect(result).to eq(Reek::CLI::Options::DEFAULT_SUCCESS_EXIT_CODE)
       end
+
+      it 'writes a todo file' do
+        command.execute
+        expected_yaml = { 'FeatureEnvy' => { 'exclude' => ['self'] } }.to_yaml
```

Comment:
Ok, I feel that at the moment we are completely misunderstanding each other.
My problem is that this:

```Ruby
{ 'FeatureEnvy' => { 'exclude' => ['self'] } }
```

is completely obscure to every reader. There is no hint, reference or clue that 'self' is something we set in our factory, buried deep down in spec/factories/factories. Additionally, when I now edit the factory in spec/factories/factories this whole spec goes booom. Which it shouldn't.

Hence I suggested to use something like this:

```Ruby
# bundle exec rake console

require 'factory_girl'
require_relative 'spec/factories/factories'

FactoryGirl.attributes_for(:smell_warning)

# => {:context=>"self", :source=>"dummy_file", :lines=>[42], :message=>"smell warning message", :parameters=>{}}
```

You see what I mean?